Makefile

```diff
@@ -0,0 +1,10 @@
+#/bin/bash
+
```

Comment:
In a very large projet (like https://github.com/mongodb/mongo-tools ), I agree, a makefile (or shell script) is useful.
I like to keep my projects a simple as possible. :)